src/Utils/Navigation/AppNavigator.cs

```diff
@@ -64,7 +64,7 @@ public Task NavigateAsync(string target, bool animated = false, bool inNewWindow
 
     private static bool AllowsInNewWindow()
     {
-        return DeviceInfo.Platform == DevicePlatform.WinUI;
+        return true || DeviceInfo.Platform == DevicePlatform.WinUI;
```

Comment:
Hi can you change the above to this ? we need it on pretty much everywhere except IOS
```cs
return DeviceInfo.Platform != DevicePlatform.IOS
```